cloudinary-core/src/main/java/com/cloudinary/Radius.java

```diff
@@ -0,0 +1,78 @@
+package com.cloudinary;
+
+import java.io.Serializable;
+
+
+public class Radius implements Serializable {
+
+private Object radius;
+    String concat = "";
+
+    public Radius() {
+    }
+
+    public Radius(Object value) {
+        if(value==null || value=="")
+            throw new IllegalArgumentException("Must supply value for the radius");
+        else {
+            this.radius = value;
+            Transformation transformation = new Transformation();
+            transformation.radius(value);
+        }
+    }
+
+    public Radius(Object topLeftAndBottomRight, Object topRightAndBottomLeft) {
+        if(topLeftAndBottomRight==null || topRightAndBottomLeft==null)
+            throw new IllegalArgumentException("Must supply two values for the radius ");
+        else{
+            radius = topLeftAndBottomRight + ":" + topRightAndBottomLeft;
+            Transformation transformation = new Transformation();
+            transformation.radius(radius);
+        }
+    }
+
+    public Radius(Object topLeft, Object topRightAndBottomLeft, Object bottomRight) {
+        if(topLeft==null || topRightAndBottomLeft==null || bottomRight==null)
+            throw new IllegalArgumentException("Must supply three values for the radius ");
+        else {
+            radius = topLeft + ":" + topRightAndBottomLeft + ":" + bottomRight;
+            Transformation transformation = new Transformation();
+            transformation.radius(radius);
+        }
+    }
+
+    public Radius(Object topLeft, Object topRight, Object bottomRight, Object bottomLeft) {
+        if(topLeft==null || topRight==null || bottomRight==null || bottomLeft==null )
+            throw new IllegalArgumentException("Must supply four values for the radius ");
+        else{
+            radius = topLeft + ":" + topRight + ":" + bottomRight + ":" + bottomLeft;
```

Comment:
@aditimadan-Cloudinary , let's avoid replicating this code. 
All "convenient methods" should eventually call the same code (maybe the Radius(Object[]) function) which should handle any number (2-4) values.